scripts/upload.gradle

```diff
@@ -128,24 +130,20 @@ afterEvaluate { project ->
         }
 
         task androidJavadocsJar(type: Jar, dependsOn: androidJavadocs) {
+            classifier "javadoc"
             from androidJavadocs.destinationDir
             baseName "${JAR_PREFIX}${project.name}${JAR_POSTFIX}"
         }
 
         task androidSourcesJar(type: Jar) {
+            classifier = 'sources'
```

Comment:
If you only have plain text you should use ' otherwise you can insert variables with "

'Hello World'
"Hello ${varName}"

I'll fix it.